site/content/docs/dashboards/_index.md

```diff
@@ -1,46 +1,41 @@
 ---
-title: Url and Custom Domain
+title: Overview
 weight: 1
 menu:
   docs:
     parent: "Dashboards"
+    identifier: overview-dashboard
 aliases:
-- "/dashboards/dashboard-url/"
-- "/docs/dashboards/dashboard-url/"
+- "/dashboards/overview/"
+- "/docs/dashboards/overview/"
 ---
 
 Checkly dashboards are public dashboards available on every plan. Use them on your wall mounted TVs, on
 shared monitors around the office or as a status page on your site.
 
-An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com).
-
 > Dashboards can be **embedded in iframes**.
 
 You can create multiple, distinct dashboards based on your plan. Edit your dashboard by clicking on
 the **Dashboards** button on the Checkly default dashboard page.
 
-## Custom URL
-
-Every public dashboard comes with a configurable custom subdomain under the checklyhq.com domain. By default, we generate
-a random ID. You can change this subdomain to anything you like as long as it is unique among all Checkly users.
-Typically a company name works best.
+An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com) as shown in the screenshot below.
 
-![custom url for public dashboard](/docs/images/dashboards/custom_url.png)
+![Dashboard example](/docs/images/dashboards/dashboard_example.png)
 
+## Adding checks to your dashboard
 
-## Custom Domain
+In order to show specific checks in a dashboard, you will need to do this:
```

Comment:
@josep-fl this whole section still reads like a support ticket, can we improve it?

Reply:
Also, let's make it clear that all checks are automatically shown if we do not specify exactly which ones to include.